src/mappings/StakeChanged.ts

```diff
@@ -75,24 +75,28 @@ export async function handleRewardRestakeForAnalytics(event: SubstrateEvent): Pr
     let {event: {data: [accountId, amount]}} = event
     let accountAddress = accountId.toString()
 
-    const payee = await cachedRewardDestination(accountAddress, event)
-    if (payee.isStaked) {
-        let amountBalance = (amount as Balance).toBigInt()
-        let accumulatedAmount = await handleAccumulatedStake(accountAddress, amountBalance)
-
-        const element = new StakeChange(eventId(event));
-        if (event.extrinsic !== undefined) {
-            element.extrinsicHash = event.extrinsic?.extrinsic.hash.toString()
+    try {
+        const payee = await cachedRewardDestination(accountAddress, event)
```

Comment:
What is the problem here? Why do we skip errors?

Reply:
maybe it's a bug or you open old version, but I rewrote this code more than week ago